MesaModel/hamilton/parameter_input/lhs_params.csv

```diff
@@ -1,5 +1,5 @@
-test_id,teacher_control,teacher_quality
-1,1,1
-2,1,0
-3,0,1
-4,0,0
\ No newline at end of file
+test_id,teacher_control_mean,teacher_quality_mean,random_select,school_learn_factor,home_learn_factor,school_learn_mean_divisor,school_learn_sd,school_learn_random_proportion,ticks_per_school_day,ticks_per_home_day,number_of_holidays,weeks_per_holiday,group_size,group_by_ability
```

Comment:
The last four parameters here are defined as user defined parameters in data_types.py. Does this suggest that they shouldn't be parameterised and if so should they be defined by LHS?

Mainly wondering this as the LHS is one of the next todos for me :)

Reply:
Ah I remember now. I think we said these will be fixed in advance of parameterisation. Therefore my LHS algorithm will write out the same values for each test_id's values for `number_of_holidays,weeks_per_holiday,group_size,group_by_ability`